### PR TITLE
docs: add onboarding calendly links to docs

### PR DESCRIPTION
**Pull request by leeederek**
Changelog picked up from commits here:

docs: add onboarding calendly to our docs banner
docs: add onboarding links to quickstart
docs: add onboarding calendly links to docs

### CODE REVIEW

File: docs/docs/home.md [hunk 1]
```diff
@@ -24,3 +24,7 @@ Kurtosis is designed to optimize **environment** management and control across t
 ![Why Kurtosis](../static/img/home/kurtosis-utility.png)
 
 With Kurtosis, developers can build with local sandbox environments that demonstrate how their code will work when integrated with the rest of the system. In addition, advanced end-to-end testing workflows are available to teams using the manipulation tooling in the Kurtosis engine runtime which allow them to do end-to-end testing like fault-tolerance, regression, and performance tests.
+
+:::info
+To schedule a live on-boarding session with us, go [here.](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```
Comment (mieubrisse) [inline docs/docs/home.md]: 1. I looked up on-boarding vs onboarding, and it looks like [the general preference is without](https://grammarhow.com/onboarding-or-on-boarding/).
1. Mind if we not include the period, to appease my OCD? 😆 (This is actually the convention we've followed across all the docs)

File: docs/docs/quickstart.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ sidebar_label: Quickstart
 slug: /quickstart
 ---
 
-These instructions will give you a brief quickstart of Kurtosis. They should take 15 minutes.
+These instructions will give you a brief quickstart of Kurtosis and should take 15 minutes. Get a personalized on-boarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding).
```
Comment (mieubrisse) [inline docs/docs/quickstart.md]: Maybe can make it flow a bit better? "If you prefer, you can get a personalized...."

File: docs/docs/quickstart.md [hunk 3]
```diff
@@ -719,6 +719,10 @@ These are just the basics of Kurtosis. To dive deeper, you can now:
 - Explore [Kurtosis-provided packages being used in production][kurtosis-managed-packages]
 - [Search GitHub for Kurtosis packages in the wild][wild-kurtosis-packages]
 
+:::info
+Get a personalized on-boarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding).
```
Comment (mieubrisse) [inline docs/docs/quickstart.md]: Same "on-boarding" comment, here and everywhere

**Review by mieubrisse — APPROVED** [review]
Approved pending some very minor things

File: docs/docs/home.md (hunk 1 above)
Comment (leeederek) [reply docs/docs/home.md]:
```suggestion
To schedule a live onboarding session with us, go [here].(https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
```

File: docs/docs/quickstart.md (hunk 3 above)
Comment (leeederek) [reply docs/docs/quickstart.md]:
```suggestion
Get a personalized onboarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 
```

File: docs/docs/quickstart.md (hunk 2 above)
Comment (leeederek) [inline docs/docs/quickstart.md]:
```suggestion
These instructions will give you a brief quickstart of Kurtosis and should take 15 minutes. If you prefer, schedule a personalized onboarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 
```

**Review by leeederek — COMMENTED** [review]
Commits to address Kevin's feedback

**Pull request merged**